blog/20240924-device-tracker-entity-description.md

```diff
@@ -21,3 +21,5 @@ The following shorthand attributes have also been added:
 - `ScannerEntity._attr_ip_address`
```

Comment:
Not sure if it matters, but the format of the date in the name of the blog file isn't consistent with the previous blogs.